packages/@sanity/default-layout/src/navbar/search/components/SearchResults.tsx

```diff
@@ -74,57 +81,68 @@ export function SearchResults({
    */
   useEffect(() => {
     setVirtualListScrollToIndex(scrollToIndex)
-    return () => setVirtualListScrollToIndex(null)
   }, [setVirtualListScrollToIndex, scrollToIndex])
 
   /**
    * Add current search terms to recent searches, trigger child item click and close search
    */
   const handleResultClick = useCallback(() => {
     // Add terms to Local Storage
-    const updatedRecentSearches = recentSearchesStore?.addSearchTerm(terms)
-    dispatch({
-      recentSearches: updatedRecentSearches,
-      type: 'RECENT_SEARCHES_SET',
-    })
+    if (recentSearchesStore) {
+      const updatedRecentSearches = recentSearchesStore.addSearchTerm(terms)
+      dispatch({
+        recentSearches: updatedRecentSearches,
+        type: 'RECENT_SEARCHES_SET',
+      })
+    }
     onChildClick?.()
     onClose()
   }, [dispatch, onChildClick, onClose, recentSearchesStore, terms])
 
   return (
-    <SearchResultsDiv aria-busy={result.loading} $loading={result.loading}>
-      <PointerOverlay ref={setPointerOverlayRef} />
+    <SearchResultsFlex direction="column">
+      {/* Sort menu */}
+      {!!result.hits.length && <SortMenu small={small} />}
```

Comment:
Slightly subjective I suppose, but I prefer to assign these sort of checks to variables such as `hasSearchResults` to be more readable, or write them as `result.hints.length > 0`.

`!!0` works fine, but I find it takes slightly more brainpower to do the "integer to boolean, then negate it" conversion in my head 🙂